File: dicrete-math/functions/javascript/02/main.js
function square(array) {
  nums = [];
  for (index = 0; index < array.length; index++) {
    nums.push(array[index] * array[index]);
  }
  return nums;
}
var numbers = [2, 3, 4, 5, 6, 7, 8, 9];
console.log(`${square(numbers)}`);

File: dicrete-math/functions/golang/02/main.go
package main

import "fmt"

func main(){
	//create an array
	arrays := []int{3, 5, 7, 9, 8, 13, 2, 11, 1}
//call the function
	fmt.Printf("The square of the array is %v\n", square(arrays))
	fmt.Printf("The cube of the array is %v\n",cube(arrays))
}
func square(numbers []int)[]int {
	var num[]int
	for index := 0; index < len(numbers); index++{
		
		num=append(num, numbers[index]*numbers[index])
	}
	return num
}

func cube(nums []int) []int{
	var num[]int
	for _,number := range nums{
		number=number*number*number
		num=append(num, number)
	}
	return num
}
